### DISCUSSION TITLE
Actividad 0 - Bravo, Diego Federico

### DISCUSSION THREAD

**Discussion by Fedebravo12** [frlp-utn-ingsoft/catedra#70] [Show and tell]
https://github.com/Fedebravo12/Actividad0.git

**Comment by RodrigoJacznik** [comment]
Hola.
Los commits están bien (aunque podrían ser mas descriptivos). La app funciona correctamente, lo único para mencionar es que en el template de detalle de una poll te quedo un for de mas, si queres lo podes arreglar (si fue intencional déjalo así).
La actividad esta aprobada.

Saludos.
